README-VALGRIND.md

```diff
@@ -0,0 +1,22 @@
+#To get valgrind output with suppressed errors
+
+(for instance with command `make`):
+
+## 1. Generate valgrind output
+####(test_bes_make.txt)
+
+`valgrind --max-stackframe=2142999 --leak-check=full --show-reachable=yes --error-limit=no --gen-suppressions=all --log-file=test_bes_make.txt make
```

Comment:
I wonder if the --max-stackframe value is arbitrary. If so, please add a note that it might not be needed or might need to be a larger value.